crates/fuel-core/src/graphql_api/database.rs

```diff
@@ -143,7 +150,7 @@ pub struct ReadView {
     pub(crate) genesis_height: BlockHeight,
     pub(crate) on_chain: OnChainView,
     pub(crate) off_chain: OffChainView,
-    pub(crate) balances_enabled: bool,
+    pub(crate) balances_indexation_enabled: bool,
```

Comment:
Maybe we also could add coins to spend here with `allow(dead_code)`?

Reply:
It's now added and used in the [complete PR](https://github.com/FuelLabs/fuel-core/pull/2463).